timelock-agent/src/main/java/com/palantir/timelock/paxos/TimeLockAgent.java

```diff
@@ -65,7 +65,7 @@
 
 @SuppressWarnings("checkstyle:FinalClass") // This is mocked internally
 public class TimeLockAgent {
-    private static final Long SCHEMA_VERSION = 1L;
+    private static final Long SCHEMA_VERSION = 2L;
```

Comment:
and this is safe because healthchecks are not reported until HTTP endpoints are opened, so we won't ever report a 2 until we start